code/control/DNRoot.php

```diff
@@ -1096,14 +1096,6 @@ public function gitRevisions(SS_HTTPRequest $request) {
 			return $this->environment404Response();
 		}
 
-		// For now only permit advanced options on one environment type, because we hacked the "full-deploy"
```

Comment:
so.. this is going to give everybody the option of 'full deploy and no rollback'?